setup.py

```diff
@@ -16,7 +16,7 @@
 from setuptools import find_packages
 from setuptools import setup
 
-version = '3.1.2'
+version = '3.1.3'
```

Comment:
Thank but there is no need to bump the version in your PR, so you can leave this out